agent/meson.build

```diff
@@ -1,28 +1,27 @@
-agents = [
-  'relay-agent',
-  'recorder-agent',
-]
+hwangsae_relay_agent_src = ['relay-agent.c']
+
+hwangsae_recorder_agent_src = ['recorder-agent.c', 'http-server.c']
 
 hwangsae_agent_c_args = [
   '-DG_LOG_DOMAIN="HWANGSAE-AGENT"',
   '-DHWANGSAE_COMPILATION',
 ]
 
-libcurl_req_version = '>= 7.0.0'
-
-libcurl_dep = dependency('libcurl', version: libcurl_req_version)
-
-foreach agent: agents
-  exe_name = 'hwangsae-@0@'.format(agent)
-  src_file = '@0@.c'.format(agent)
+soup_dep = dependency('libsoup-2.4')
+json_glib_dep = dependency('json-glib-1.0')
```

Comment:
If I can have one more small request, pls move the dependencies to the toplevel `meson.build`. :)